plugin/templates/_eslint.config.mjs

```diff
@@ -0,0 +1,9 @@
+import pluginJs from "@eslint/js";
+import pluginNode from "eslint-plugin-n";
+import eslintPlugin from "eslint-plugin-eslint-plugin";
+
+export default [
+    pluginJs.configs.recommended,
+    pluginNode.configs["flat/recommended"],
```

Comment:
I ran `yo eslint:plugin` to create a plugin and it was successful.

But, running `npm run lint:js` in the plugin reports:

```
C:\projects\tmp\tmp\eslint.config.mjs
  1:1  error  Parsing error: 'import' and 'export' may appear only with 'sourceType: module'

✖ 1 problem (1 error, 0 warnings)
```

Reply:
fixed in ed00e494a3444de515a6bc1421390207d515e990